doc/source/manual/xtrabackup_sst.rst

```diff
@@ -343,9 +348,57 @@ The ``tmpdir`` option can be set in the following :file:`my.cnf` groups:
   (if not specified under ``[sst]``)
 * ``[mysqld]`` is used if it is not specified in either of the above
 
+.. option:: wsrep_debug
+
+   :Version: Introduced in 5.7.19-29.22
+   :Default: ``OFF``
+
+Specifies whether additional debugging output for the database server error log
+should be enabled.
+Disabled by default.
+
+This option can be set in the following :file:`my.cnf` groups:
+
+* Under ``[mysqld]`` it enables debug logging for ``mysqld`` and the SST script
+* Under ``[sst]`` it enables debug logging for the SST script only
+
+.. option:: encrypt_threads
+
+   :Version: Introduced in 5.7.19-29.22
+   :Default: ``4``
+
+Specifies the number of threads that XtraBackup should use for encrypting data
+(when ``encrypt=1``).
+The value is passed using the ``--encrypt-threads`` option in XtraBackup.
+
+This option affects only SST with XtraBackup
+and should be specified under the ``[sst]`` group.
+
+.. option:: backup_threads
+
+   :Version: Introduced in 5.7.19-29.22
+   :Default: ``4``
+
+Specifies the number of threads that XtraBackup should use to create backups.
+See the ``--parallel`` option in XtraBackup.
+
+This option affects only SST with XtraBackup
+and should be specified under the ``[sst]`` group.
+
 XtraBackup SST Dependencies
 ---------------------------
 
+Although any current version of |PXB| *may* be compatible
```

Comment:
David suggested a text in his recent email do you want to use that.

Reply:
"forever married" is not quite the wording for formal documents. I rephrased it to properly highlight that we require a specific version because that's how we test it, not because there are actual incompatibilities.